nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -179,15 +170,21 @@ let cfg = config.fonts.fontconfig;
       ln -s ${renderConf}       $support_folder/conf.d/10-nixos-rendering.conf
       ln -s ${renderConf}       $latest_folder/conf.d/10-nixos-rendering.conf
 
-      # 30-nixos-generic-alias.conf
-      ln -s ${genericAliasConf} $support_folder/conf.d/30-nixos-generic-alias.conf
-      ln -s ${genericAliasConf} $latest_folder/conf.d/30-nixos-generic-alias.conf
+      # 50-user.conf
+      ${optionalString (! cfg.includeUserConf) ''
+      rm    $support_folder/conf.d/50-user.conf
+      rm    $latest_folder/conf.d/50-user.conf
+      ''}
 
-      # 99-user.conf
-      ${optionalString cfg.includeUserConf ''
-      ln -s ${userConf}         $support_folder/conf.d/99-user.conf
-      ln -s ${userConf}         $latest_folder/conf.d/99-user.conf
+      # local.conf (indirect priority 51)
+      ${optionalString (cfg.localConf != "") ''
+      ln -s ${localConf}        $support_folder/local.conf
+      ln -s ${localConf}        $latest_folder/local.conf
```

Comment:
> Weren't the versioned configurations only put in place for backwards compatibility?

Right, I must confess that I don't know much about the reasons and implications for multiple fontconfig version support so I might have made silly assumptions.

I just thought that there might be a need to have different `local.conf` for each fontconfig version in case there were incompatibilities.

But the main reason is that, as a personal taste, I think it easier to understand and cleaner to have symmetric file structure for both versions.

But I have no problem in removing `latest/local.conf` in favor of a single `/etc/fonts/local.conf` if it is more appropriate.